cvmfs/cache.cc

```diff
@@ -636,8 +636,10 @@ int64_t PosixCacheManager::Write(const void *buf, uint64_t size, void *txn) {
   Transaction *transaction = reinterpret_cast<Transaction *>(txn);
 
   if (transaction->expected_size != kSizeUnknown) {
-    if (transaction->size + size > transaction->expected_size)
+    if (transaction->size + size > transaction->expected_size) {
```

Comment:
I'm not entirely sure why it's nested.  It was nested before my modification, but I will modify it.